### PR TITLE
updated error message

### PR DESCRIPTION
**Pull request by cs-pranjali-munfan**
PR title: updated error message 

Description: updated error message in case of invalid username and password provided.

Impact: check health

UTC:
- install the connector using .tgz
- verified error message for invalid credentials.

### CODE REVIEW

File: fidelis-edr/operations.py [hunk 1]
```diff
@@ -89,6 +89,8 @@ def get_authorisation_token(self):
             return response.get('data').get('token')
         else:
             logger.exception(str(response))
+            if 'Bad username or password' in str(response):
+                raise ConnectorError('Invalid endpoint or credentials')
```
Comment (cs-suyog-jinturkar) [inline fidelis-edr/operations.py:93]: Why this change is needed?

Comment (cs-pranjali-munfan) [reply fidelis-edr/operations.py:93]: to make a consistent error message for most of the connectors. 
Bug - https://mantis.fortinet.com/bug_view_page.php?bug_id=0863405

Comment (cs-suyog-jinturkar) [reply fidelis-edr/operations.py:93]: okey. got it.
